samples/glsl_flat_smooth_interpolators.html

```diff
@@ -145,43 +145,52 @@
         var programs = [createProgram(gl, getShaderSource('vs-flat'), getShaderSource('fs-flat')), createProgram(gl, getShaderSource('vs-smooth'), getShaderSource('fs-smooth'))];
         var uniformMvpLocations = [gl.getUniformLocation(programs[VIEWPORTS.LEFT], "mvp"), gl.getUniformLocation(programs[VIEWPORTS.RIGHT], "mvp")];
         var uniformMvNormalLocations = [gl.getUniformLocation(programs[VIEWPORTS.LEFT], "mvNormal"), gl.getUniformLocation(programs[VIEWPORTS.RIGHT], "mvNormal")];
-        
-        var vertexPosBuffer = gl.createBuffer();
-        var vertexNormalBuffer = gl.createBuffer();
+
+        var vertexBuffer = gl.createBuffer();
         var vertexElementBuffer = gl.createBuffer();
         var vertexArray = gl.createVertexArray();
 
-        // -- Load obj then render
-        var objURL = '../assets/obj/di_model_tri.obj';
-        loadObj(objURL, function(mesh) {
+        // -- Load gltf then render
+        var relativePath = '../assets/gltf/';
+        var filename = 'di_model_tri.gltf';
+        loadGltf(relativePath, filename, function(scene) {
```

Comment:
A cleaner interface is to get the relative path from the full path passed in (e.g., `../assets/gltf/di_model_tri.gltf`) and also allow an optional parameter to override the relative path.  For example, see https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Core/getBaseUri.js